.circleci/config.yml

```diff
@@ -301,6 +301,22 @@ workflows:
                 path: cypress/results
           requires:
             - PR_UNIT_TESTS
+      - cypress/run:
+          pre-steps:
+            - run: 'rm -rf ~/.yarn && npm i -g yarn && yarn -v' # Use yarn latest
+          yarn: true
+          store_artifacts: true
+          build: yarn run build:package && cd platform/viewer && yarn run make-script-tag-example
+          start: yarn run start-server-script-tag
+          wait-on: 'http://localhost:3000'
```

Comment:
This is already doing the magic that `start-server-and-test` is supposed to be buying us.

Reply:
That is true, but you can't run the CircleCI orb locally, so I think we still want to include this so that the e2e tests can be run locally.